spec/features/create_comment.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'User comments', type: :feature do
+  before :each do
+    User.create(name: 'mike', email: 'mike@gmail.com', password: '111111')
+    Post.create(content: 'text', author_id: 1)
+  end
+
+  scenario 'has comment content' do
+    login_with 'mike@gmail.com', '111111'
+    fill_in 'comment_content', with: 'Good Day'
```

Comment:
**[REQUIRED]**
This line is causing this error while running the tests.
```
Failure/Error: fill_in 'comment_content', with: 'Good Day'
     
     Capybara::ElementNotFound:
       Unable to find field "comment_content" that is not disabled
```

Reply:
We resolved this as well